src/gen/flags.rs

```diff
@@ -20,6 +20,10 @@ pub const ENABLE_ASSERT_BEFORE: u32 = 0x100000;
 // disallow relative height- and time conditions on ephemeral spends
 pub const NO_RELATIVE_CONDITIONS_ON_EPHEMERAL: u32 = 0x200000;
 
+// use the generator ROM that does not pass in the clvm parser as the first
+// argument
+pub const HARD_FORK: u32 = 0x400000;
```

Comment:
Here I think the name should definitely be different. A hard fork may end up having other changes beyond the generator. So this flag should probably be `USE_V2_GENERATOR_ROM` or something like that, ie. just the hard fork features that are relevant at this scope.

Reply:
it's a good point. the name should indicate the actual change it makes, and if there are multiple changes as part of the hard fork, they should have different flags